pom.xml

```diff
@@ -19,11 +19,11 @@
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
     <version>3.37</version>
-    <relativePath/>
+    <relativePath />
   </parent>
 
   <artifactId>google-kubernetes-engine</artifactId>
-  <version>0.3.1-SNAPSHOT</version>
+  <version>0.4.1-SNAPSHOT</version>
```

Comment:
I think this should be 0.4.0-SNAPSHOT so that when the release command runs it will update to 0.4.1-SNAPSHOT

Reply:
This is the result of the release command's work :)

Reply:
Didn't look at the commit history, my bad.